diff --git a/CONFIG/src/probe_comp.c b/CONFIG/src/probe_comp.c
index 4cdf8d4..861b60e 100644
--- a/CONFIG/src/probe_comp.c
+++ b/CONFIG/src/probe_comp.c
@@ -613,12 +613,14 @@ char *GetPtrbitsFlag(enum OSTYPE OS, enum MACHTYPE arch, int ptrbits,
       sp = (ptrbits == 64) ? "-maix64" : "-maix32";
    else if ((MachIsX86(arch) || MachIsPPC(arch) || MachIsUS(arch)) ||
             arch == IbmPwr7 || arch == IbmPwr6 || arch == Pwre6500)
+#ifndef __arm__
    {
       if (ptrbits == 64)
          sp = "-m64";
       else if (ptrbits == 32)
          sp = "-m32";
    }
+#endif
    return(sp);
 }
 char *GetStandardCompName(char *comp, int f77)
--- ATLAS/CONFIG/src/SpewMakeInc.c.orig	2015-04-11 10:03:47.177469952 +0100
+++ ATLAS/CONFIG/src/SpewMakeInc.c	2015-04-11 10:05:05.030464216 +0100
@@ -420,12 +420,14 @@
       sp = (ptrbits == 64) ? "-maix64" : "-maix32";
    else if ((MachIsX86(arch) || MachIsPPC(arch) || MachIsUS(arch)) ||
             arch == IbmPwr7 || arch == IbmPwr6 || arch == Pwre6500)
+#ifndef __arm__
    {
       if (ptrbits == 64)
          sp = "-m64";
       else if (ptrbits == 32)
          sp = "-m32";
    }
+#endif
    return(sp);
 }
 
